test/integration/interrupt_resume_test.rb

```diff
@@ -77,8 +77,8 @@ def test_interrupt_resume_with_writes_to_source
     assert_test_table_is_identical
   end
 
-  def test_interrupt_resume_when_table_has_completed
-    # Start a run of Ghostferry expecting to be interrupted
+  def test_interrupt_ignored_when_table_has_completed
```

Comment:
> Ghostferry::Status::ROW_COPY_COMPLETED

I assume that state maps to `StateCutover` in the main Ferry code?

Reply:
Yes, ROW_COPY_COMPLETED is received after data copy is done and ferry state has just been set to `StateCutover`: https://github.com/Shopify/ghostferry/blob/d2df94173c6deea7d99d463cb9bc17b98c183129/ferry.go#L634